docs/processing_information.md

```diff
@@ -71,6 +71,19 @@ Finally, these principal components are used to calculate the [UMAP (Uniform Man
 
 [Graph-based clustering](http://bioconductor.org/books/3.16/OSCA.basic/clustering.html#clustering-graph) is also performed, using the Louvain algorithm with 20 nearest neighbors and Jaccard weighting.
 
+#### Cell type annotation
+
+We additional perform cell type annotation with two complementary methods, where possible:
+
+- [`SingleR`](https://bioconductor.org/packages/release/bioc/html/SingleR.html), a reference-based cell type annotation method
+- [`CellAssign`](https://github.com/Irrationone/cellassign), a marker-gene--based cell type annotation method
+
+For `SingleR` annotation, we identify an appropriate reference dataset from the [`celldex` package](http://bioconductor.org/packages/release/data/experiment/html/celldex.html) and train the classification model to use ontology IDs for annotation.
+Cells which `SingleR` cannot be confidently are given `NA` annotations.
+
+For `CellAssign` annotation, we identify an appropriate set of marker genes for the given tissue type from the [`PanglaoDB`](https://panglaodb.se/) database.
```

Comment:
We are also combining these with an immune organ, so I think we want to point that out too.